server/index.js

```diff
@@ -0,0 +1,5 @@
+const app = require('./server');
+
+const port = process.env.PORT || 3000;
+
+app.listen(port, () => console.log(`Express now departing from port ${port}!`))
```

Comment:
could put "http://localhost:" instead of "port" for easy access to server on browser but not major